workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandRequestSender.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Improbable.Gdk.Core.Commands;
+using Improbable.Worker;
+using Unity.Entities;
+using UnityEngine.Assertions.Must;
+using Entity = Unity.Entities.Entity;
+
+#region Diagnostic control
+
+// ReSharper disable UnusedMember.Local
+
+#endregion
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static partial class WorldCommandsRequirables
+    {
+        [InjectableId(InjectableType.WorldCommandRequestSender, InjectableId.NullComponentId)]
```

Comment:
This one could be tied to the world command sender component. Not really much in it though.

Reply:
Misunderstood. Ignore this.